src/Api/AdminConsole/Models/Request/Organizations/OrganizationCreateRequestModel.cs

```diff
@@ -14,42 +14,56 @@ public class OrganizationCreateRequestModel : IValidatableObject
     [StringLength(50, ErrorMessage = "The field Name exceeds the maximum length.")]
     [JsonConverter(typeof(HtmlEncodingStringConverter))]
     public string Name { get; set; }
+
     [StringLength(50, ErrorMessage = "The field Business Name exceeds the maximum length.")]
     [JsonConverter(typeof(HtmlEncodingStringConverter))]
     public string BusinessName { get; set; }
+
     [Required]
     [StringLength(256)]
     [EmailAddress]
     public string BillingEmail { get; set; }
+
     public PlanType PlanType { get; set; }
-    [Required]
-    public string Key { get; set; }
+
+    [Required] public string Key { get; set; }
+
     public OrganizationKeysRequestModel Keys { get; set; }
     public PaymentMethodType? PaymentMethodType { get; set; }
     public string PaymentToken { get; set; }
-    [Range(0, int.MaxValue)]
-    public int AdditionalSeats { get; set; }
-    [Range(0, 99)]
-    public short? AdditionalStorageGb { get; set; }
+
+    [Range(0, int.MaxValue)] public int AdditionalSeats { get; set; }
+
+    [Range(0, 99)] public short? AdditionalStorageGb { get; set; }
+
     public bool PremiumAccessAddon { get; set; }
+
     [EncryptedString]
     [EncryptedStringLength(1000)]
     public string CollectionName { get; set; }
+
     public string TaxIdNumber { get; set; }
+
     public string BillingAddressLine1 { get; set; }
+
     public string BillingAddressLine2 { get; set; }
+
     public string BillingAddressCity { get; set; }
+
     public string BillingAddressState { get; set; }
+
     public string BillingAddressPostalCode { get; set; }
-    [StringLength(2)]
-    public string BillingAddressCountry { get; set; }
+
+    [StringLength(2)] public string BillingAddressCountry { get; set; }
```

Comment:
⛏️ Optional: Please break a line on the data annotations so the format is the same as everywhere else